bigtable_input.py

```diff
@@ -349,15 +376,24 @@ def get_unparsed_moves_from_last_n_games(n,
       A dataset containing no more than `moves` examples, sampled
         randomly from the last `n` games in the table.
     """
-    _games.wait_for_fresh_games()
-    latest_game = int(_games.latest_game_number())
-    utils.dbg('Latest game: %s' % latest_game)
-    if latest_game == 0:
-        raise ValueError('Cannot find a latest game in the table')
-
-    start = int(max(0, latest_game - n))
-    ds = _games.moves_from_games(start, latest_game, moves, shuffle,
-                                 column_family, column)
+    c_r, c_nr = 9, 1
```

Comment:
i assume these are 'count', 'take', and 'percent', but a one line comment would aid legibility

Reply:
I agree. Will do.